eventClass/eventHandler.py

```diff
@@ -1,5 +1,7 @@
 from collections import deque
-from aeroCubeEvent import AeroCubeEvent
+from .aeroCubeEvent import AeroCubeEvent, ResultEvent
+from .aeroCubeSignal import *
+from enum import Enum
 
 
 class EventHandler(object):

```

Comment:
Class definitely needs more documentation to clarify its behavior (in terms of the different states and how state transitions should occur, both publicly and internally).